luigi_pipeline/lib/model/sv_mt_schema.py

```diff
@@ -203,16 +205,11 @@ def _genotype_fields(self):
             'num_alt': self._num_alt(is_called)
         }
 
-    # NB: override this function here to mimic the existing null handling behavior.
-    def _genotype_filter_samples(self, filter_):
-        samples = super()._genotype_filter_samples(filter_)
-        return hl.or_missing(hl.len(samples) > 0, samples)            
-
     @row_annotation(name="samples_gq_sv", fn_require=SeqrGenotypesSchema.genotypes)
     def samples_gq(self):
         # NB: super().samples_gq is a RowAnnotation... so we call the method under the hood.
         # ew it is gross.
-        return super().samples_gq.fn(self, start=0, end=1000, step=10)
+        return super().samples_gq.fn(self, start=0, end=100, step=10)
```

Comment:
The inherited function `ends` at `95`... we want a 90->100 bucket right?  

Should `step` be `5`?

Reply:
step should still be 10, and no actually we do not need 90_100, seqr does searches by querying for "quality is not in these low bins" so you never need the top bin, which svaes a lot of space

Reply:
👍 ... though just calling out this deviates from previous behavior: https://github.com/broadinstitute/seqr-loading-pipelines/blob/4d0eb3486e14da9664a1c00810e395e0dbf6bfd3/sv_pipeline/genome/load_data_tests.py#L150